front50-api/src/main/java/com/netflix/spinnaker/front50/api/model/pipeline/Pipeline.java

```diff
@@ -75,30 +70,24 @@ public void setLastModifiedBy(String lastModifiedBy) {
     super.put("lastModifiedBy", lastModifiedBy);
   }
 
-  @JsonIgnore
   public Object getConfig() {
     return super.get("config");
   }
 
-  @JsonIgnore
   public void setConfig(Object config) {
     super.put("config", config);
   }
 
-  @JsonIgnore
   public String getType() {
     return (String) super.get("type");
   }
 
-  @JsonIgnore
   public void setType(String type) {
     super.put("type", type);
   }
 
-  @JsonIgnore
   public Collection<Trigger> getTriggers() {
-    return MAPPER.convertValue(
```

Comment:
Moved the deserialization of Triggers to `TriggerDeserializer` so we can remove jackson dependency: https://github.com/mochacat/front50/pull/1/files#diff-31cdddef45d43a2d1c4a40149811e5fc5460374a499076b92ac301e0aa7cae89